pkg/sql/sem/builtins/pg_builtins.go

```diff
@@ -1145,6 +1157,74 @@ SELECT description
 		},
 	),
 
+	"pg_relation_is_updatable": makeBuiltin(
+		defProps(),
+		tree.Overload{
+			Types: tree.ArgTypes{{"reloid", types.Oid}, {"include_triggers", types.Bool}},
+			ReturnType: tree.FixedReturnType(types.Int4),
+			Fn: func(ctx *tree.EvalContext, args tree.Datums) (tree.Datum, error) {
+				oidArg := tree.MustBeDOid(args[0])
+				oid := int(oidArg.DInt)
+				table, err := ctx.Planner.GetImmutableTableInterfaceByID(ctx.Ctx(), oid)
+				if err != nil {
+					return zero, nil
+				}
+				tableDesc, ok := table.(catalog.TableDescriptor)
+				if !ok || !tableDesc.IsTable() || tableDesc.IsVirtualTable() {
+					return zero, nil
+				}
+
+				return allEvents, nil
+			},
+			Info: "pg_relation_is_updatable was created for compatibility. This " +
```

Comment:
nit: you can avoid using `+` for strings by using ``` ` ```. i think we should omit the `allEvents` for now and leave it as a comment to allEvents above.

tl;dr - let's make the info:

```
Info: `"Returns the update events the relation supports.`
```